dips/dip-10.md

```diff
@@ -54,18 +54,11 @@ Example: `alice@avasp`
 * `vasp_domain_identifier` is a unique string that is mapped to a VASP. Specification:
```

Comment:
@aylinaykol / @sunmilee I think we should definitely consider a "." here so that we can support domains and not be completely disjoint from paystring (food for thought)

Reply:
okie i agree i added .

Reply:
Should it be `^[a-zA-Z0-9][a-zA-Z0-9.]*$` or `[a-zA-Z0-9.]+`  ?